### ISSUE TITLE
Elastix import error: PyCapsule_Import "_ITKCommonPython._C_API" is not valid

### ISSUE THREAD

**Issue by ablot** [InsightSoftwareConsortium/ITKElastix#159]
I tried fresh install of `itk-elastix` on two machines (mac and linux) and in both cases I ran into the same import error:

```(itk_venv) [blota@int002 venv]$ python
Python 3.9.6 (default, Nov 25 2021, 18:34:53) 
[GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import itk
>>> itk.elastix_registration_method
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/camp/lab/znamenskiyp/home/users/blota/venv/itk_venv/lib/python3.9/site-packages/itk/support/lazy.py", line 138, in __getattribute__
    base.itk_load_swig_module(module, namespace)
  File "/camp/lab/znamenskiyp/home/users/blota/venv/itk_venv/lib/python3.9/site-packages/itk/support/base.py", line 132, in itk_load_swig_module
    l_module = loader.load(swig_module_name)
  File "/camp/lab/znamenskiyp/home/users/blota/venv/itk_venv/lib/python3.9/site-packages/itk/support/base.py", line 291, in load
    l_spec.loader.exec_module(l_module)  # pytype: disable=attribute-error
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/camp/lab/znamenskiyp/home/users/blota/venv/itk_venv/lib/python3.9/site-packages/itk/support/../ElastixPython.py", line 13, in <module>
    from . import _ElastixPython
AttributeError: PyCapsule_Import "_ITKCommonPython._C_API" is not valid
```

I thought it might be linked to #154, since I originally used conda but a new venv had the same issue. Should I install something else before running `pip install itk-elastik`?

**Comment by devphilno** [comment]
Any solutions to this problem ?

**Comment by dzenanz** [comment]
@tbirdso, are you looking into this?

**Comment by tbirdso** [comment]
Hi all, the problem is a known issue in the recent `itk==v5.3rc04.post2` package release tracked in [https://github.com/InsightSoftwareConsortium/ITK/issues/3528](https://github.com/InsightSoftwareConsortium/ITK/issues/3528). A temporary workaround is to downgrade to `itk==v5.3rc04.post1` or earlier for working with current `itk-elastix` packages.

**Comment by devphilno** [comment]
worked !

**Comment by Erotemic** [comment]
The above workaround doesn't seem to be available on linux with Python 3.10. Hoping to get a proper fix soon.

**Comment by cameron-a-johnson** [comment]
NP, and here you go:
```
$ pip list | grep itk
itk                     5.3rc4.post2
itk-core                5.3rc4.post2
itk-elastix             0.14.1
itk-filtering           5.3rc4.post2
itk-io                  5.3rc4.post2
itk-numerics            5.3rc4.post2
itk-registration        5.3rc4.post2
itk-segmentation        5.3rc4.post2
```

**Comment by tbirdso** [comment]
@cameron-a-johnson What output did you get when you tried downgrading to an earlier ITK per the workaround above?

```
python -m pip install itk==v5.3rc04
```

**Comment by cameron-a-johnson** [comment]
Oh this actually worked for me! Apologies, I had only tried `itk==v5.3rc04.post1` earlier, and not any earlier versions than that. In that case I got the following output:

```
$ python -m pip install itk==v5.3rc04.post1
ERROR: Could not find a version that satisfies the requirement itk==v5.3rc04.post1 (from versions: 4.12.0.post1, 4.13.0, 4.13.1.post1, 4.13.2, 4.13.2.post1, 5.0a1, 5.0a2, 5.0b1, 5.0b3, 5.0rc1, 5.0rc2, 5.0.0, 5.0.0.post1, 5.0.1, 5.1b1, 5.1rc1.post1, 5.1rc2, 5.1rc3, 5.1.0, 5.1.0.post1, 5.1.0.post2, 5.1.0.post3, 5.1.1, 5.1.1.post1, 5.1.2, 5.2rc1, 5.2rc2, 5.2rc3, 5.2.0, 5.2.0.post1, 5.2.0.post2, 5.2.0.post3, 5.2.1, 5.2.1.post1, 5.3rc1, 5.3rc2, 5.3rc2.post1, 5.3rc3, 5.3rc4, 5.3rc4.post2)
ERROR: No matching distribution found for itk==v5.3rc04.post1
```

FYI @Erotemic

**Comment by tbirdso** [comment]
Great, glad to hear that there is still a workaround for now. I also just tried installing `post1` and it looks like there is an issue with the PyPI distribution, will edit my original comment accordingly. This issue is still under development pending a resolution in the main ITK repo.

**Comment by thewtex** [comment]
Addressed in `itk-elastix 0.14..2` thanks to @tbirdso

**Comment by tbirdso** [comment]
Thanks for tagging `0.14.2` @thewtex 👍
